src/inmanta/protocol/methods_v2.py

```diff
@@ -392,6 +392,39 @@ def get_api_docs(format: Optional[ApiDocsFormat] = ApiDocsFormat.swagger) -> Ret
     """
 
 
+@typedmethod(
+    path="/scheduler_status",
+    operation="GET",
+    arg_options=methods.ENV_OPTS,
+    client_types=[ClientType.api],
+    api_version=2,
+)
+def get_scheduler_status(tid: uuid.UUID) -> model.SchedulerStatusReport:
+    """
+    Inspect the scheduler state from the given environment.
+
+    :param tid: The id of the environment in which to inspect the scheduler.
+
+    """
+
+
+@typedmethod(
+    path="/scheduler/state",
+    operation="GET",
+    server_agent=True,
+    enforce_auth=False,
+    arg_options=methods.ENV_OPTS,
+    client_types=[],
+    api_version=2,
+)
+def trigger_get_status(tid: uuid.UUID) -> model.SchedulerStatusReport:
+    """
+    Get a snapshot of the scheduler state
+
+    :param tid: The id of the environment.
+    """
```

Comment:
I don't quite get the difference between these two endpoints

Reply:
`get_scheduler_status` is the outer one: api on the server to request the status, (the orchestration service responds to this one) and it in turns calls the inner `trigger_get_status` to retrieve the status (the new agent responds to that one)